ui/apps/platform/src/Containers/Policies/Wizard/Step2/PolicyBehaviorForm.tsx

```diff
@@ -238,36 +244,40 @@ function PolicyBehaviorForm({ hasActiveViolations }: PolicyBehaviorFormProps) {
                             component="p"
                         />
                     )}
-                    <FormGroup
-                        fieldId="policy-event-source"
-                        label="Event sources (Runtime lifecycle only)"
-                        isRequired={hasRuntime}
-                        className="pf-v5-u-pt-lg"
-                    >
-                        <Flex direction={{ default: 'row' }}>
-                            <Radio
-                                label="Deployment"
-                                isChecked={values.eventSource === 'DEPLOYMENT_EVENT'}
-                                id="policy-event-source-deployment"
-                                name="eventSource"
-                                onChange={() => setFieldValue('eventSource', 'DEPLOYMENT_EVENT')}
-                                isDisabled={!hasRuntime || hasActiveViolations}
-                            />
-                            <Radio
-                                label="Audit logs"
-                                isChecked={values.eventSource === 'AUDIT_LOG_EVENT'}
-                                id="policy-event-source-audit-logs"
-                                name="eventSource"
-                                onChange={onChangeAuditLogEventSource}
-                                isDisabled={!hasRuntime || hasActiveViolations}
-                            />
-                        </Flex>
-                        <FormHelperText>
-                            <HelperText>
-                                <HelperTextItem>{eventSourceHelperText}</HelperTextItem>
-                            </HelperText>
-                        </FormHelperText>
-                    </FormGroup>
+                    {isRuntime && (
+                        <FormGroup
+                            fieldId="policy-event-source"
+                            label="Event sources (Runtime lifecycle only)"
+                            isRequired={isRuntime}
+                            className="pf-v5-u-pt-lg"
+                        >
+                            <Flex direction={{ default: 'row' }}>
+                                <Radio
+                                    label="Deployment"
+                                    isChecked={values.eventSource === 'DEPLOYMENT_EVENT'}
+                                    id="policy-event-source-deployment"
+                                    name="eventSource"
+                                    onChange={() =>
+                                        setFieldValue('eventSource', 'DEPLOYMENT_EVENT')
+                                    }
+                                    isDisabled={!isRuntime || hasActiveViolations}
```

Comment:
Rereading this because a moment of mistaken deja vu after I had just moved to review of #16940

What is pro and con to remove `!isRuntime` I am thinking better to keep because who knows if the decision is reverted suddenly and then we (if the we was me) might forget to add it back.